src/inverseRobot.js

```diff
@@ -8,6 +8,19 @@
 
 function inverseRobot(robot) {
   // write code here
+  const robotValues = Object.values(robot);
+  const uniqueRobotValues = new Set(robotValues);
+  const inversedRobot = {};
+
+  if (robotValues.length !== uniqueRobotValues.size) {
+    return null;
+  }
+
+  for (const key in robot) {
+    inversedRobot[robot[key]] = key;
+  }
```

Comment:
You do it with out additional operations
```suggestion
  const inversedRobot = {};

  for (const key in robot) {
    const newKey= robot[key];
    
    if (inversedRobot[newKey] !== undefined) {
      return null;
    }
    
    inversedRobot[newKey] = key;
  }
```